packages/big-design/src/components/Box/Box.tsx

```diff
@@ -17,8 +17,16 @@ export interface BoxProps extends React.HTMLAttributes<HTMLDivElement>, DisplayP
   borderRadius?: keyof BorderRadius;
 }
 
-export const Box: React.FC<BoxProps> = memo(
-  React.forwardRef<HTMLDivElement, BoxProps>((props, ref) => <StyledBox forwardedRef={ref} {...props} />),
+interface PrivateProps {
+  forwardedRef: React.Ref<HTMLDivElement>;
+}
+
+const RawBox: React.FC<BoxProps & PrivateProps> = memo(
```

Comment:
There is no need to `forwardRef` here if you are already forwarding when declaring `Box`. 

Also the order should be `memo -> forwardRef`.